README.md

```diff
@@ -7,29 +7,16 @@ First check wether the file for the provider already exists, if it does skip to
 Add [provider name].md to the _providers folder in the following format:
 ~~~
 ---
+website: [website of the provider]
 name: [provider name]
-domains: [ email domains of the provider / can also be yaml array when there are multiple ones can contain (€) to mark a premium domain]
-credentials: [emailPass | emailAppPass | Oauth] - can also be an array, if multiple are supported availible
+domains: [email domains of the provider / must be an yaml array]
+before_login_hint: [~ or a string that should be displayed before the user logs in]
+after_login_hint: [~ or a string that should be displayed after the user logged in]
```

Comment:
why not just leave this field out if it is not needed? i expect that to be true for the vast majority of providers.

Reply:
I would ask for the code to set this field to the absent value (e.g. `NULL`) in any of these cases:

* The field is not present,
* the value is `~`,
* the value is an empty string or a string that contains only whitespace.

Reply:
I used serde yaml for parsing, it serialized this empty field to `~`, I didn't try leaving it out yet to see whether It'll be parsed as `None` or if it complains instead when the key is missing.

> the value is an empty string or a string that contains only whitespace.

that would require manual parsing, but we can add a check that prevents the build/ci from getting green in this case instead.

Reply:
Ok, just tested it, works also fine if the field is not present.